ui/mirage/handlers/clients.js

```diff
@@ -119,7 +121,7 @@ function generateMonths(startDate, endDate, namespaces) {
   return months;
 }
 
-function generateActivityResponse(namespaces, startDate, endDate) {
+export function generateActivityResponse(namespaces, startDate, endDate) {
```

Comment:
Since these are functions that just pertain to generating mock data for mirage, I left them in this file and exported.